lambdas/services/create_document_reference_service.py

```diff
@@ -287,7 +292,16 @@ def remove_records_of_failed_lloyd_george_upload(
 
         logger.info("Deleting dynamodb record...")
         self.document_service.hard_delete_metadata_records(
-            table_name=self.lg_dynamo_table, document_references=failed_upload_records
+            table_name=table_name, document_references=failed_upload_records
         )
 
         logger.info("Previous failed records are deleted.")
+
+    def fetch_incomplete_arf_upload_records(
+        self, nhs_number
+    ) -> list[DocumentReference]:
+        return self.document_service.fetch_available_document_references_by_type(
+            nhs_number=nhs_number,
+            doc_type=SupportedDocumentTypes.ARF,
+            query_filter=UploadIncomplete,
+        )
```

Comment:
Can we change this to `fetch_incomplete_records` and add a parameter for doc_type, and then call this fun in `check_existing_lloyd_george_records`?